io/net/htx_nic_devices.py.data/htx_nic_devices.yaml

```diff
@@ -0,0 +1,9 @@
+mdt_file: 'net.mdt'
+host_ip: x.xx.xxx.xx
```

Comment:
this is public ip, which we need to update in both host & peer machines on the file (/etc/hosts & bpt files)
host_interfaces are can be any interfaces where we execute the htx stress, and these interfaces are different than the main interface(Which will have the public IP).